.github/workflows/phpunit-tests.yml

```diff
@@ -0,0 +1,275 @@
+name: PHPUnit Tests
+
+on:
+  # Anytime a branch or tag is created.
+  create:
```

Comment:
We should probably look into whether this is actually necessary since the appearance of a new branch/tag on the remote should also be a push event.

Reply:
I removed this for now. If we find new branches don't trigger workflows we can revisit!